code/WorkInProgress/TempEngine.dm

```diff
@@ -155,14 +155,14 @@
 
 		handle_reactions(removed)
 
-		// Leaks gas varient
+		// Leaks gas variant
 		if((circulator_flags & LEAKS_GAS ) && prob(5))
 			var/datum/gas_mixture/leaked = gas_input.remove_ratio(rand(2,8)*0.01)
 			src.audible_message("<span class='alert'>[src] makes a hissing sound.</span>")
 			if(leaked) loc.assume_air(leaked)
 
-		if(src.network1) src.network1.update = 1
-		if(src.network2) src.network2.update = 1
+		src?.network1.update = 1
+		src?.network2.update = 1
```

Comment:
not equivalent, you want `src.network1?.x`

Reply:
🤦